test/components/converters/test_pypdf_to_document.py

```diff
@@ -7,74 +7,80 @@
 from haystack.dataclasses import ByteStream
 
 
-@pytest.mark.integration
```

Comment:
Based on my understanding, I would classify tests that require opening and converting a file as integration tests, even if they are fast. (Each of these should be marked `@pytest.mark.integration`)

However, some of the following tests do not seem integration tests to me: ` test_init`,  
 `test_init_fail_nonexisting_converter`, `test_run_with_meta`,  `test_run_error_handling`. (I would not mark these).

Reply:
Okay sounds good, I'll go ahead and mark the ones that involve opening a file as integration tests.